MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -364,6 +365,12 @@ private void listItems(int id) {
           new Intent(MainActivity.this, HillShadeActivity.class),
           R.string.activity_style_hillshade_url, false, BuildConfig.MIN_SDK_VERSION
         ));
+        exampleItemModels.add(new ExampleItemModel(
+          R.string.activity_styles_fade_switch_title,
+          R.string.activity_styles_fade_switch_description,
+          new Intent(MainActivity.this, StyleFadeSwitchActivity.class),
+          R.string.activity_styles_fade_switch_url, false, BuildConfig.MIN_SDK_VERSION
```

Comment:
This is a new example right, we should be setting this to true if so?

Reply:
Fixed